runners/spark/src/test/java/org/apache/beam/runners/spark/io/NumShardsTest.java

```diff
@@ -21,10 +21,10 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
-import org.apache.beam.examples.WordCount;
```

Comment:
No it doesn't, it's just been like that... Not a good answer 😄 but I'm definitely prioritising.